### PR TITLE
feat: allowlist of allowed github owners

### PR DESCRIPTION
**Pull request by c-bata**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -24,6 +24,13 @@ const (
 	maxSize = 5 * 1024 * 1024
 )
 
+var (
+	// A whitelist of GitHub owners who allowed to use this proxy.
```
Comment (orisano) [inline main.go]: whitelist -> allowlist

File: main.go [hunk 2]
```diff
@@ -24,6 +24,13 @@ const (
...
+	// You can set multiple owners via environment variables like:
+	// PROXY_ALLOWED_OWNERS="orisano:c-bata"
+	allowedOwners map[string]interface{}
```
Comment (orisano) [inline main.go]: Please `allowedOwners` to be UnzipHander's field.

Comment (orisano) [inline main.go]: `map[string]interface{}` -> `map[string]struct{}`

File: main.go [hunk 3]
```diff
@@ -86,6 +103,15 @@ func (h *UnzipHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	tokens := strings.SplitN(req.URL.Path, "/", 6)
 	owner := tokens[1]
 	repo := tokens[2]
+
+	if allowedOwners != nil {
```
Comment (orisano) [inline main.go]: If the map is nil, ok is always false, so this if is not necessary

**Review by orisano — CHANGES_REQUESTED** [review]

**Comment by c-bata** [comment]
Thank you for your review! PTAL.

**Review by orisano — APPROVED** [review]
LGTM!

**Comment by orisano** [comment]
@c-bata thanks for your contribution!

**Pull request merged**